src/js/messaging/components/compose/MessageSubjectGroup.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import classNames from 'classnames';
+import MessageCategory from './MessageCategory';
+import MessageSubject from './MessageSubject';
+
+class MessageSubjectGroup extends React.Component {
+  // TODO: Add errorMessage property to ErrorableTextInput conditionally
+  // when the fields are validated.
+  render() {
+    const hasErrorMsg = !!this.props.errorMessage;
+
+    const errItemClass = classNames(
+      this.props.cssErrorClass,
+      'msg-subject-group',
+      { 'usa-input-error': hasErrorMsg },
+      { 'msg-compose-error': hasErrorMsg }
+    );
+
+    const errMsgClass = classNames(
+      { 'msg-compose-error-message': hasErrorMsg },
+      { 'usa-input-error-message': hasErrorMsg },
+    );
```

Comment:
This might as well go inside the conditional below this. Since `hasErrorMsg` is definitely true within that block, the class names there can either (1) be written without the conditional object notation or (2) inline with the `div`.